next-sitemap.config.js

```diff
@@ -0,0 +1,18 @@
+// next-sitemap.config.js
+
+/** @type {import('next-sitemap').IConfig} */
+module.exports = {
+  siteUrl: 'https://chained-assets.vercel.app',
```

Comment:
should it be `https://chained-assets.vercel.app` or `https://chainedassets.com` ?

Reply:
it should be https://chainedassets.com. at that time i donot know that we also deployed this website on another url as well